### ISSUE TITLE
Nova's phone player UI does not rotate video at all; stays at 90 degrees counterclockwise position in portrait mode

### ISSUE THREAD

**Issue by Pentaphon** [nova-video-player/aos-AVP#461]
This is easy to reproduce. Moto G 2021 stock ROM, Nova 5.15.20

1) Play video on your phone in portrait mode. 
2) Note that video is playing in the orientation that is correct when you turn it 90 degrees counterclockwise. 
3) Turn phone 180 degrees. 
4) See that video is upside down even though it should conveniently flip. 
5) Note that "enable automatic screen rotation" is checked but video doesn't rotate. 

This isn't an issue on other video apps like Newpipe, which rotate the video according to the way the phone is rotated, even if the phone's portrait mode is enabled.

**Comment by courville** [comment]
It works on m'y side. Have you tried enabling autorotate in phone quick
settings?

**Comment by Pentaphon** [comment]
> It works on m'y side. Have you tried enabling autorotate in phone quick settings? 

Yes, it works when I enable autorotate but I always use portrait because I find autorotate very annoying and video apps like YouTube and Newpipe still rotate in portrait mode because otherwise your video would be upside down if you rotate your phone for any reason such as having to plug it in which forces you to turn the phone the other way, etc.  Is there any way to make the video always rotate even if you are on portrait mode like Newpipe does? You say you don't see this behavior. Which phone are you using? I just tried playing a video on a Samsung tablet with stock Android 11 in portrait mode and Nova wouldn't flip the video when I turned the tablet, but then I played a video on Newpipe and it flipped the video to the right orientation when I turned the tablet. 

I realize this isn't a major issue but if you can allow Nova to flip the video even when using portrait mode the way Newpipe does, it would make Nova more convenient to use.

**Comment by courville** [comment]
OK I see your point now.
There are two autorotate options: one in Android quick setting in notif bar and one in nova player menu.
In your specific case, I understand you deactivate Android autorotate.
If in nova player menu you deactivate nova autorotate option, it should force landscape video play mode always.
Would this meet the expected behavior?

**Comment by Pentaphon** [comment]
> If in nova player menu you deactivate nova autorotate option, it should force landscape video play mode always.
> Would this meet the expected behavior?

Whether I uncheck "enable automatic screen rotation" or not, the video never flips upside down when I turn the phone the other way. It always plays in landscape and only when you hold the phone 90 degrees counterclockwise. Here's what I mean. Let's pretend the following images are Nova playing a video. 

Here's how Nova plays a video on my phone. You turn your phone counterclockwise 90 degrees to see the video in the right orientation. 

![nova](https://user-images.githubusercontent.com/49374058/129706553-a487b69f-0e2b-49aa-841a-d75428747f1a.jpg)

When I turn the phone 180 degrees clockwise for a reason like plugging it in to charge it while watching a video, I expect the video to flip, even in portrait mode, so I can see the video the right way but all I get is this. An upside-down video, because Nova doesn't compensate. 

![nova2](https://user-images.githubusercontent.com/49374058/129707078-e008823c-568a-41da-8a35-8d9bd9298e2f.jpg)

What I want is for Nova to "sense" that the video is playing upside down, even when Android's portrait mode is enabled, and flip the video so it plays in the right orientation. Both Youtube and Newpipe do this just fine.
